strongbox-distribution/src/main/resources/postinstall

```diff
@@ -0,0 +1,3 @@
+groupadd strongbox
+useradd -d /usr/local/strongbox -g strongbox strongbox
+chown -R strongbox:strongbox /usr/local/strongbox/
```

Comment:
Please, make sure you add new lines to the end of files when creating, or editing them, as this can sometimes be a problem on different operating systems.